perceval/utils/logging/loggers.py

```diff
@@ -137,7 +137,7 @@ def warn(self, msg: str, channel: exq_log.channel = DEFAULT_CHANNEL):
     def _format_exception(self, exc_info=None) -> str:
         if not exc_info:
             return ""
-        return ': '+' '.join(traceback.format_exception(exc_info[0], exc_info[1], exc_info[2])).replace("\n", " ").replace("    ", " ").replace("  ", ", ")
+        return '\n' + ''.join(traceback.format_exception(exc_info[0], exc_info[1], exc_info[2]))
```

Comment:
This will create a weird log entry and will add a multi line entry, it will be a pain if the log are parsed line by line.

Reply:
It's a normal readable callstack